samples/batch/batch-ingestion-client/Setup/guide.md

```diff
@@ -41,21 +41,35 @@ If the above link does not work try the following steps:
 4. Click Create on the Speech resource.
 5. You will find the subscription key under **Keys**
 6. You will also need the region, so make a note of that too.
+7. You need to decide on the operating mode [read next section] 
 
-To test your account we suggest you use [Microsoft Azure Storage Explorer](https://azure.microsoft.com/features/storage-explorer/).
+To test your the set up we reccomend you use [Microsoft Azure Storage Explorer](https://azure.microsoft.com/features/storage-explorer/).
 
-### The Project
+### Operating Mode
 
-Although you do not need to download or do any changes to the code you can still download it from GitHub:
+Audio files can be processed either by the v3 API (batch) or our Speech SDK (real time). Both operations are wrapped and implemented in serverless and although seamless to the user there are sublte differences. The goal of this section is to explain the pros and cons of these options.
 
-```
-git clone https://github.com/Azure-Samples/cognitive-services-speech-sdk
-cd cognitive-services-speech-sdk/samples/batch/transcription-enabled-storage
-```
+#### Batch Mode
+
+In this mode you will experience the following
```

Comment:
Maybe add a sentence about the mode instead of "In this mode you will experience the following"